app/models/proposal.rb

```diff
@@ -13,7 +13,7 @@ class Proposal < ActiveRecord::Base
 
   # The following list also servers as an interface spec for client_datas
   delegate :fields_for_display, :client, :public_identifier, :total_price,
-           :name, to: :client_data_legacy
+           :name, :version, to: :client_data_legacy
```

Comment:
What do you think about changing it to this?

```ruby
def version
  [
    self.updated_at,
    self.client_data_legacy.updated_at
  ].max.to_i
end
```

There might be unintended consequences I'm not thinking of...